code/go/0chain.net/smartcontract/dbs/event/search.go

```diff
@@ -0,0 +1,40 @@
+package event
+
+import (
+	"database/sql"
+	"strconv"
+)
+
+// GetGenericSearchType returns kind of search query
+func (edb *EventDb) GetGenericSearchType(query string) (string, error) {
```

Comment:
`query` is something like the entire command in sql domain..
maybe we should use a better name `searchString`

Reply:
Yes, I agree, renamed the param!